### ISSUE TITLE
Can't interact

### ISSUE THREAD

**Issue by nkleinert** [nefarius/Indicium-Supra#8]
Hello, I can't interact with ImgUi.
Exist any fix? (I used the unchanged example plugin)

Example:
![Example](https://i.imgur.com/wlzHfD1.gif)

**Comment by nefarius** [comment]
Yep, I recently fixed that in a different branch, I'll back-port it and then you can test.

**Comment by nefarius** [comment]
Pushed, please have a look.

**Comment by xcxooxl** [comment]
I'm still having this same problem, Is something changed of lately|?

**Comment by nefarius** [comment]
@xcxooxl that heavily depends on the game. It's just a reference implementation, not a universal solution.
